### PR TITLE
Refactor styled components

### PR DESCRIPTION
**Pull request by fyang1024**
I did initial refactoring -- put those reusable styled components in one place for reuse. It's tested.

But there seems to be a lot more to do:
1. We probably need to turn `color` in IAppSettings into `colorScheme` object which includes colors to render the button, etc and the styled buttons will need to read those colors dynamically.
2. We repeat 311px 327px many times in different components. We probably come up with a simpler box model, e.g, a 311px wide container for everything in it. that way, when we change the width, we change one place.

Since there are already a lot of changes, I'll leave those changes for next pull request

### CODE REVIEW

File: src/ts/components/account/GeneratePhrase.tsx [hunk 1]
```diff
@@ -9,13 +8,14 @@ import { Message, Container, Grid, Button, Icon } from 'semantic-ui-react'
 import { generateMnemonic } from '../../services/keyring-vault-proxy'
 import { setNewPhrase } from '../../background/store/account'
 import { CONFIRM_PHRASE_ROUTE } from '../../constants/routes'
+import { Button as StyledButton, Section, MnemonicPad } from '../basic-components'
```
Comment (ttaoS) [inline src/ts/components/account/GeneratePhrase.tsx]: why we do  `Button as StyledButton` here?

Comment (ttaoS) [reply src/ts/components/account/GeneratePhrase.tsx]: don't worry. I see why now

File: src/ts/components/basic-components.ts [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  font-weight: 800;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: 1.31;
+  letter-spacing: normal;
+  text-align: center;
+  color: #ffffff;
+`
+
+export const Section = styled.p`
```
Comment (mytechtip) [inline src/ts/components/basic-components.ts]: better to use `div` instead of `p` as section will contain other components

Comment (fyang1024) [reply src/ts/components/basic-components.ts]: agreed. updated and pushed

**Pull request merged**